cf_xarray/accessor.py

```diff
@@ -203,18 +208,19 @@ def _apply_single_mapper(mapper):
     return results
 
 
-def _get_axis_coord_single(var: Union[DataArray, Dataset], key: str) -> List[str]:
-    """ Helper method for when we really want only one result per key. """
-    results = _get_axis_coord(var, key)
-    if len(results) > 1:
-        raise KeyError(
-            f"Multiple results for {key!r} found: {results!r}. I expected only one."
-        )
-    elif len(results) == 0:
-        raise KeyError(f"No results found for {key!r}.")
-    return results
+def _set_doc(doc):
```

Comment:
hmm.. i think this is a little too complicated. Can we just add the docstrings for the functions directly instead? I think the redundancy is OK, I don't expect it to change...